website/screens/components/date-input/specs/DateInputSpecsPage.tsx

```diff
@@ -0,0 +1,521 @@
+              <td>
+                <Code>pickerCurrentDateColor</Code>
+              </td>
+              <td>Dialog title</td>
+              <td>
+                <Code>color-grey-500</Code>
+              </td>
+              <td>#999999</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerHoverDateBackgroundColor</Code>
+              </td>
+              <td>Container background</td>
+              <td>
+                <Code>color-purple-200</Code>
+              </td>
+              <td>#e5d5f6</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerSelectedDateColor</Code>
+              </td>
+              <td>Date text:selected</td>
+              <td>
+                <Code>color-white</Code>
+              </td>
+              <td>#ffffff</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerSelectedDateBackgroundColor</Code>
+              </td>
+              <td>Date background:hover</td>
+              <td>
+                <Code>color-purple-700</Code>
+              </td>
+              <td>#5f249f</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerHoverDateFontColor</Code>
+              </td>
+              <td>Date text:hover</td>
+              <td>
+                <Code>color-black</Code>
+              </td>
+              <td>#000000</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerYearFontColor</Code>
+              </td>
+              <td>Year label</td>
+              <td>
+                <Code>color-black</Code>
+              </td>
+              <td>#000000</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerMonthFontColor</Code>
+              </td>
+              <td>Mont label</td>
+              <td>
+                <Code>color-black</Code>
+              </td>
+              <td>#000000</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerWeekFontColor</Code>
+              </td>
+              <td>Weekday label</td>
+              <td>
+                <Code>color-black</Code>
+              </td>
+              <td>#000000</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerMonthArrowsBackgroundColor</Code>
+              </td>
+              <td>Month selection controls</td>
+              <td>
+                <Code>color-transparent</Code>
+              </td>
+              <td>transparent</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>pickerFocusColor</Code>
+              </td>
+              <td>Focus indicator</td>
+              <td>
+                <Code>color-blue-600</Code>
+              </td>
+              <td>#0095ff</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+      </DxcStack>
+      <DxcStack gutter="large">
```

Comment:
It is not necessary to add a stack for each table. A stack for the whole design tokens section is enough.

Reply:
Done